vuepress/docs/next/tutorials/devops/caching-and-clustering.md

```diff
@@ -144,76 +144,43 @@ git checkout tags/v6.3.68 -b 6.3.2-redis ****Check
 6. Build and push a custom image for the `entando-de-app` with [Building a Docker Image Tutorial](./build-core-image.md).
 7. Download the operator configuration deployment file:
 
-``` bash
-curl -L -C - -O https://raw.githubusercontent.com/entando/entando-releases/v7.0.0/dist/ge-1-1-6/namespace-scoped-deployment/orig/namespace-resources.yaml > namespace-resources.yaml
+``` sh
+curl -L -C - -O https://raw.githubusercontent.com/entando/entando-releases/v7.0.1/dist/ge-1-1-6/namespace-scoped-deployment/orig/namespace-resources.yaml > namespace-resources.yaml
 ```
 
+8. Update the image for `entando-de-app-wildfly` in the `namespace-resources.yaml` file to point to your custom `entando-de-app` image with Redis. The line to change is in the `ConfigMap` named `entando-docker-image-info`.
 
-8. Update the image in the deployment YAML file to point to your custom `entando-de-app` image with Redis. The line to change is in the `ConfigMap` and noted below:
-
+9. Deploy your edited file with `kubectl`.
 
-```
-entando-de-app-wildfly: >-
-    {"version":"6.3.10","executable-type":"jvm","registry":"docker.io","organization":"entando"}
-```
-
-9. Deploy your edited file with `kubectl`. For example:
-
-```
+```sh
 kubectl apply -f namespace-resources.yaml
 ```
 
-<!-- 
-10. Run the Helm template to generate an application configuration:
-=======
-
-```
-helm template quickstart ./ > my-clustered-app.yaml
-```
-
-11. Add environment variables to the `EntandoApp` in the outcome of the Helm template command generated above (`my-clustered-app.yaml`) for the Redis address and Redis password for your deployed Redis instance. The variables to create are: -->
-
-```
-REDIS_ADDRESS
-```
-```
-REDIS_PASSWORD
-```
-```
-REDIS_ACTIVE
+10. Download the `entando-app.yaml` template
+```sh
+curl -sLO "https://raw.githubusercontent.com/entando/entando-releases/v7.0.1/dist/ge-1-1-6/samples/entando-app.yaml"
 ```
 
-**The following example pertains to the EntandoApp only, not a complete deployment.** Utilize this as a reference to create your configuration in a complete deployment.
+11. Add environment variables to the `EntandoApp` defition to match your Redis instance. The variables to create are REDIS_ACTIVE, REDIS_ADDRESS (e.g. _redis://localhost:6379_), and REDIS_PASSWORD.
```

Comment:
was just about to approve, but was reading through and found typo: definition. Otherwise approved